x/auth/stdtx_test.go

```diff
@@ -51,3 +55,58 @@ func TestStdSignBytes(t *testing.T) {
 		require.Equal(t, tc.want, got, "Got unexpected result on test case i: %d", i)
 	}
 }
+
+func TestTxValidateBasic(t *testing.T) {
```

Comment:
Can we add a test for too many signatures now that we added that check?

Reply:
Done :)